google-cloud-firestore/src/test/java/com/google/cloud/firestore/it/ITQueryTest.java

```diff
@@ -28,12 +28,8 @@
 import com.google.cloud.firestore.*;
 import com.google.cloud.firestore.Query.Direction;
 import java.time.Duration;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.Iterator;
-import java.util.LinkedHashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
The Java style guide [disallows](https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports) wildcard imports (`import foo.*`). Although sadly this hasn't been respected in a few places (such as line 28 above).